crates/asm/txs/bridge-v1/src/deposit_request/descriptor/format.rs

```diff
@@ -0,0 +1,355 @@
+//! Deposit descriptor wire format used by bridge deposits.
+//!
+//! A descriptor says "route this deposit to account serial X and subject bytes Y". The encoding is
+//! compact: it avoids zero padding in the subject and uses fewer bytes for small serial values.
+//!
+//! Layout (all values are big-endian):
+//! `[control:1][serial:1..=3][subject:0..=32]`
+//!
+//! Control byte (bit 7 is MSB):
+//! - bits 7..6: reserved, must be 0
+//! - bits 5..4: serial length (00=1 byte, 01=2 bytes, 10=3 bytes, 11 reserved)
```

Comment:
see the next comment: it could be that we don't need any of these, because serial fit in 4 bits (this is what I'd expect for quite some time) -- seems to me we'd have to use `00` here just to have a full-zero byte.